applications/targets/tests/test_target_create.py

```diff
@@ -0,0 +1,89 @@
+import factory
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+from rest_framework.views import status
+
+from .factories import TargetFactory, TopicFactory
+from applications.users.tests.factories import UserFactory
+from applications.targets.models import Target
+
+
+class TargetCreateTest(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.url = reverse('target-list')
+
+    def call_create_target(self):
+        return self.client.post(self.url, self.params)
+
```

Comment:
I would move this after the `setUp`, I think it's more readable taking into account that this is referring to `self.url` and `self.params`